### PR TITLE
Ambiguous filters exact

### PR DESCRIPTION
**Pull request by KillWil**
Refers to [issue 466](https://github.com/spatie/laravel-query-builder/issues/466)

### CODE REVIEW

**Comment by lexdewilligen** [comment]
Please merge this! Really need it

**Comment by nickpoulos** [comment]
Just checking in on the status of this PR? We are also experiencing the same issue.

File: src/Filters/FiltersExact.php [hunk 1]
```diff
@@ -29,12 +29,12 @@ public function __invoke(Builder $query, $value, string $property)
         }
 
         if (is_array($value)) {
-            $query->whereIn($property, $value);
+            $query->whereIn($query->getModel()->getTable().'.'.$property, $value);
```
Comment (BrandonSurowiec) [inline src/Filters/FiltersExact.php:32]:
```suggestion
    $query->whereIn($query->qualifyColumn($property), $value);
```

**Comment by AlexVanderbist** [comment]
Hi, thank you for reminding me. Tests for this PR do seem to be failing, I'll look into this right now.

Personally I'm still on the fence about this. I understand that combining joins with filters causes issues but this is also true for basic Eloquent. This package isn't meant to solve these kind of issues, it's just meant as a way to access Eloquent from the URL.
For example, the problem described in the issue (#466) could be fixed by specifying the table in the underlying `where` like this: `Model::query()->join('other_models', 'other_models.code', 'models.code)->where('other_models.code', 'foo')`. This directly applies to this package as well: `GET /models?filters[other_models.code]=foo`.

**Pull request merged**